cmake/external_gtest.cmake

```diff
@@ -24,22 +24,9 @@ include(ExternalProject)
 SET(GTEST_GIT_REPO_URL https://github.com/google/googletest.git)
 SET(GTEST_GIT_LABEL release-1.8.1)
 
-if (CMAKE_CXX_COMPILER_ID STREQUAL "GNU")
-    if (DEFINED NGRAPH_USE_CXX_ABI)
-        set(COMPILE_FLAGS "${COMPILE_FLAGS} -D_GLIBCXX_USE_CXX11_ABI=${NGRAPH_USE_CXX_ABI}")
```

Comment:
Don't we still need to set the ABI flag somewhere?

Reply:
https://github.com/NervanaSystems/ngraph/blob/master/CMakeLists.txt#L318-L323 already sets it in CMAKE_CXX_FLAGS

Reply:
And https://github.com/NervanaSystems/ngraph/pull/2942/files/056b0e2eeff308677e9242b4094a6e851f824b58#diff-2abb742095220ead92ffaef7701a3ec3R51 passes CMAKE_CXX_FLAGS down to external_gtest